pages/news/2020-08-20-Meter-Detection.html

```diff
@@ -0,0 +1,41 @@
+title: Experimental Meter Detection Code
+author: Cristiano Lacerda
+date: 2020-08-20 12:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+The meter detection code is an extension of the tempogram [as described by Peter Grosche, Meinard Muller and Frank Kurth](http://resources.mpi-inf.mpg.de/MIR/tempogramtoolbox/2010_GroscheMuellerKurth_TempogramCyclic_ICASSP.pdf) and is based on the QM Vamp plugin that implement such features. 
```

Comment:
We need an introduction here.
This is just a suggestion, feel free to write a better introduction instead:

```suggestion
One of the parts where Mixxx is lacking is how it handles time signatures and downbeats.
Currently, Mixxx only has individual beats but has no support for detecting or displaying additional information about the rhythm.
 
This is going to change during this year's [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/), because we have two students working on detecting and on displaying rhythm information.
This post focuses on the implementation of meter detection, which is a chance to put Mixxx ahead of some of its commercial competitors that just assume a 4/4 time signature for all tracks. 

The meter detection code is an extension of the tempogram [as described by Peter Grosche, Meinard Muller and Frank Kurth](http://resources.mpi-inf.mpg.de/MIR/tempogramtoolbox/2010_GroscheMuellerKurth_TempogramCyclic_ICASSP.pdf) and is based on the QM Vamp plugin that implement such features. 
```